whois-commons/src/main/java/net/ripe/db/whois/common/rpsl/transform/FilterAuthFunction.java

```diff
@@ -105,6 +118,15 @@ public RpslObject apply(final RpslObject rpslObject) {
         }
     }
 
+    private boolean canAuthenticate(final List<RpslAttribute> attributes){
+        return !attributes.isEmpty() || override != null;
+    }
```

Comment:
Why we just need to check for override, not for password or any other auth methods?

Reply:
let say this
if canAuthenticate we check if isMntnerAuthenticated or isOverrideAuthenticated

So the credentials comes later, in canAuthenticate we only check if it has the requirements.

Reply:
I think it is confusing to check here.  

if authAttributes are empty but overrideCredentials are not , it will return true? 
In that case it should return false , no? 

Also just checking  overrideCredentials != null is not enough, you need to check overrideValues is present or not ?

Reply:
is it working as expected,
Override doesn't have any kind of relation with the auth attributes. This is not happening for the rest of authentications

Reply:
Only these four conditions can happen 

    // !true || true) == true (why ??) there are no auth attributes so why authenticate
    // !true )  == false
    // !false ) == true
    // !false ) == true

Reply:
yes, the name of the class is FilterAuthAttributes...so if there is no one then we return doesn't matter what.
Sorry for the confusion, you are totally right I reverted the change